Source/Scene/Cesium3DTileFeature.js

```diff
@@ -55,6 +53,24 @@ define([
          * @private
          */
         this.primitive = tileset;
+
+        /**
+         * The <code>tile</code> containing this feature.
+         *
+         * @type {Cesium3DTile}
+         *
+         * @private
+         */
+        this.tile = content._tile;
```

Comment:
Also, are we sure we need this property?  There will be **a lot** of these objects since they are per-feature and we want to keep them as small as possible.